extensions/noteplan-3/src/lib/note-utilities.ts

```diff
@@ -25,8 +30,16 @@ export interface Note {
   content: string;
 }
 
+const getPreferences = () => {
+  return getPreferenceValues<Preferences>();
+};
```

Comment:
I guess you could simply use 
```suggestion
export const { preferences } = getPreferenceValues<Preferences>()
```
here, or do you need it to be a closure?